GomokuServer/src/GomokurServer.Core/Entities/Game.cs

```diff
@@ -6,34 +6,28 @@ namespace GomokuServer.Core.Entities;
 public class Game
 {
 	private readonly List<GameMove> _playersMoves = new();
+	private readonly GameBoard _gameBoard;
 
-	public string GameId { get; } = Guid.NewGuid().ToString();
+	public Game(GameBoard gameBoard)
+	{
+		_gameBoard = gameBoard;
```

Comment:
Can we please put validation for board property? In case, if I accidentally pushed 5 million rows, what happens?

Reply:
What max board size you want?

Reply:
Okay. I see in wikipedia, that gomoku usually is played on 15*15 or 19*19 board. Let's put 19 as max board size?